environment.yaml

```
@@ -10,15 +10,11 @@ dependencies:
 - xarray
```

```suggestion
- xarray
- dask
- netCDF4
- bottleneck
```

Adding xarray's recommended dependencies